package/src/components/ChannelList/ChannelList.tsx

```diff
@@ -314,12 +314,13 @@ export const ChannelList = <
     setChannels,
   });
 
-  const channelIdsStr = channels.reduce((acc, channel) => `${acc}${channel.cid}`, '');
+  const channelIdsStr = channels?.reduce((acc, channel) => `${acc}${channel.cid}`, '');
 
   useEffect(() => {
-    if (staticChannelsActive || !enableOfflineSupport) {
+    if (channels === null || !enableOfflineSupport) {
```

Comment:
Small doubt, is it fine to remove the check of `staticChannelsActive` from here?

Reply:
Good catch. This will cause un-necessary updates to DB.

Reply:
Fixed!!